docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -217,24 +224,60 @@ Unlike agent-based models, microsimulations do not necessarily program agency in
 
 .. todo:: 
   
-  Expand upon how microsim is different from agent-based modeling here. 
+  -  The Vivarium simulation with the most simulant agency is our synthetic population simulation for PRL. [explain how in 1 sentence]
+  - HPVsim (along with others in Starsim series) provides example of agent-based simulation (eg different simulants have different sexual patterns leading to different risks)
+  - Cite definition of agent-based simulation 
+
 
 Incorporating uncertainty
```

Comment:
Should this section have its title changed to match the content? Maybe "Reducing variance between scenarios"?

Reply:
I like that change!